### ISSUE TITLE
Add React root element support

### ISSUE THREAD

**Issue by Vladi4-gh** [jantimon/html-webpack-plugin#1411]
This is madness to find an extra plugin like https://www.npmjs.com/package/html-webpack-partials-plugin or https://www.npmjs.com/package/html-webpack-root-plugin (doesn't work) to just add a simple root div!

**Comment by Shaun2D2** [comment]
I'm running into the same frustration and i have to create either a template or use an outside package to get this functionality.  Would the project maintainers be on board with this happening?  I would be happy to submit a PR to get this support if it is in scope of the package.

**Comment by jantimon** [comment]
I am not sure how we can implement this feature in a good way to support not only react but also other frameworks and to work well not only with the default template but also with custom templates.

Right now you can do the following - why would that be not a good way for you?

```
new HtmlWebpackPlugin({
  templateContent: `
    <html>
      <body>
        <div id="app-root"></div>
      </body>
    </html>
  `
})
```

**Comment by Vladi4-gh** [comment]
Well, it was unclear for me that templateContent will be populated with scripts and styles and etc. I've already used a html-webpack-partials-plugin to add a root div. However, I just tried to use a template property, seems like I'll be OK with this one. I just created the html I really need and populated it with props from <%= htmlWebpackPlugin.files.* %> and it generated correctly. 
Well, seems like I can close this thread, thanks for your attention.
